ui/src/components/item.js

```diff
@@ -58,31 +80,112 @@ class Item extends React.Component{
         }
     }
 
+    placeBid=()=>{
+        if(this.state.jwt==""){
+            toast.error('Have to be logged in to place bids', {
+                position: "top-center",
+                autoClose: 3000,
+                hideProgressBar: true,
+                closeOnClick: true,
+                pauseOnHover: false,
+                draggable: false,
+                progress: 0,
+            });
```

Comment:
since there are multiple configs that are same, add some utility constant in /utils/constants.js named DEFAULT_TOAST_CONFIG

or something similar